runner/src/server/plugins/engine/views/components/clientsidefileuploadfield.html

```diff
@@ -261,9 +272,48 @@
         errorSpan.textContent = errorMessage;
     };
 
+    // Show error summery with the error message in js side
+    const handleErrorSummery = (errorMessage) => {
```

Comment:
```suggestion
    // Show error summary with the error message in js side
    const handleErrorSummary = (errorMessage) => {
```

Reply:
Just a spelling error on this one and across the function/pr

Reply:
Fixed the issue